templates/python/custom-copilot-rag-azure-ai-search/infra/azure.parameters.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
do we need to add `.tpl` suffix to this file?

Reply:
oh yeah~ sure ofcourse

Reply:
added